.travis.yml

```diff
@@ -7,12 +7,12 @@ php:
   - hhvm
 
 env:
-  - SYMFONY_VERSION=2.6.*
+  - SYMFONY_VERSION="~2.3 || ~3.0"
```

Comment:
that doesn't seem to be proper composer syntax:
``The requested package || could not be found, it looks like its name is invalid, "||" is not allowed in package names.``